tests/src/test/scala/it/unibo/scafi/test/functional/ScafiTestUtils.scala

```diff
@@ -89,4 +93,9 @@ object ScafiTestUtils {
     val res = expr
     if(pred(res)) res else until(pred)(expr)
   }
+
+  def standardNetwork(): Network with SimulatorOps = {
```

Comment:
Better a name `gridNetwork` than `standardNetwork`

Reply:
I think `gridNetwork` may be misleading because the last node is disconnected from the others

Reply:
Still

- it is "almost" a `gridNetwork`, and
- `standardNetwork` is even worse

Reply:
Moreover, I am a bit unsure about defining the network in a separate place than the test class.
The reader of test code should be able to immediately find the test fixture (i.e., how the network), despite code duplication.
We could provide factory/builder methods for "recurrent" networks, but then we must use clear names such as `manhattanNet3x3WithSWdetached` where `manhattan` describes connectivity (left/right/top/bottom), `3x3` the size, and after `With` other peculiar features (e.g., that `S`outh`W`est device is detached)

Reply:
I'll add a network builder with a clear name